### DISCUSSION TITLE
Zoom with font-size from dev tools on Firefox

### DISCUSSION THREAD

**Discussion by hwillson** [apollographql/apollo-client-devtools#384] [Ideas]
Migrated from: https://github.com/apollographql/apollo-client-devtools/issues/252

Thank you for publishing this very helpful package.

When I zoom Firefox Dev tools' font size, the apollo pane does not zoom in with me

Repro:

1. Side-load the dev tools into firefox accd to https://github.com/apollographql/apollo-client-devtools/issues/73#issuecomment-542604418
2. load up a site with apollo client hooked in to dev tools
3. open dev tools and switch to apollo tab
4. zoom with <kbd>cmd</kbd>+<kbd>+</kbd>

Result: FF Dev tools font size increases (i.e. the tab bar) but Apollo dev tools font-size does not:

I did not have this same result on Chrome, where the font size scales with the window
 
<img width="1300" alt="Screen Shot 2020-04-02 at 19 17 10" src="https://user-images.githubusercontent.com/1466420/78273217-37dab880-7517-11ea-96b3-980ae361c4c5.png">

**Comment by benatshippabo** [comment]
I like this idea! Firefox doesn't get as much love as Chrome these days. Although I think this may be a Firefox issue, since I have the same issues with any third party devtool on Firefox

**Comment by alessbell** [comment]
Confirming I can still reproduce this.
